backend/aggregations/app.mjs

```diff
@@ -113,7 +130,63 @@ app.get("/revenue-query", function (req, res) {
 
   const newQuery = generator.stringify(parsedQuery);
 
-  console.log("starting query...\n", newQuery);
+  query(newQuery)
+    .then(function (response) {
+      res.send(
+        JSON.stringify({
+          results: response.results.bindings,
+        })
+      );
+    })
+    .catch(function (err) {
+      res.send(JSON.stringify(err));
+    });
+});
+
+app.get("/decisions-query", function (req, res) {
+  // Use current year as default
+  const year = req.query.year || new Date().getFullYear();
+  const municipality = req.query.municipality;
+
+  if (typeof year !== "number" && year.length != 4) {
```

Comment:
I think `year` will be a string by default, so you would need to check if it can be converted to a string, not if it already is a string.

I thiiiink the easiest way would be to check if `parseInt(year) !== NaN`.  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
There's a similar thing on line 32, but I'm not sure about that one. JS types without typescript are scary :p

Reply:
I have a middleware that parses numbers by default (and other fancy types)

Reply:
Ooooooo fancyy, nice!